builder/dockerfile/builder.go

```diff
@@ -151,19 +151,23 @@ func (b *Builder) build(source builder.Source, dockerfile *parser.Result) (*buil
 	addNodesForLabelOption(dockerfile.AST, b.options.Labels)
 
 	if err := checkDispatchDockerfile(dockerfile.AST); err != nil {
+		triggeredBuilds.WithValues("error_parsing_dockerfile").Inc()
```

Comment:
@stevvooe, yep, makes sense. I've split the `triggered_builds` into 3 metrics: `triggeredBuilds, triggeredFailedBuilds and triggeredSuccessBuilds`. It's going to be simpler and that seems to be recommended approach used by the `prometheus` community.